specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```diff
@@ -6943,6 +6943,92 @@
       },
       "description": "Base type for skills."
     },
+    "CustomEntity": {
+      "properties": {
```

Comment:
Will any of these be sent back in a response as explicit `null`? If so, they must be marked `"x-nullable": true`.

Reply:
Except "name", all other parameters are optional. Per previous discussion, we should save bandwidth to not use "x-nullable" here.

Reply:
But I bet you send them back with explicit nulls, which breaks the deserializers. The point of saving bandwidth was that you NOT send back values with explicit nulls. Putting x-nullable in the swagger - which isn't transmitted with every request - saves no bandwidth.

Last I heard from @brjohnstmsft it was the OData returning values with explicit nulls. If you can't turn that off, you need to mark any optional parameters with x-nullable: true.